geopandas/geodataframe.py

```diff
@@ -1320,14 +1321,21 @@ def to_parquet(
             The encoding to use for the geometry columns. Defaults to "WKB"
             for maximum interoperability. Specify "geoarrow" to use one of the
             native GeoArrow-based single-geometry type encodings.
-        schema_version : {'0.1.0', '0.4.0', '1.0.0', None}
-            GeoParquet specification version; if not provided will default to
-            latest supported version.
+            Note: the "geoarrow" option is part of the upcoming GeoParquet 1.1
```

Comment:
One downside of writing "upcoming" is that this will of course be outdated very quickly .. Maybe I should just say a more generic "newer"?

Reply:
If we remembered we could update it when updating `latest supported stable version (1.0.0)` below, but switching to newer is probably safer.